### DISCUSSION TITLE
KEDA operator to scale pods at pod level to reduce carbon emission

### DISCUSSION THREAD

**Discussion by zhifanl** [kedacore/keda#5710] [Ideas]
Hello everyone!
I’m exploring the research with a professor about developing a specialized KEDA operator focused on low-carbon impact, aimed at enabling more precise scaling decisions at the pod level. The concept revolves around enhancing efficiency in scaling individual pods up or down based on specific metrics and labels, potentially improving the carbon footprint of our operations.
The core idea is to allow fine-grained control over which pods are scaled, based on a combination of custom-defined metrics and sustainability criteria.

I know k8s typically manages scaling at the level of deployments or ReplicaSets rather than individual pods.

I am currently doing some research about if there is such existing tools, if you have expertise in this pod level scaling, can anyone provide me some advice or anything? any advice will be appreciated. thanks.

**Comment by zroubalik** [comment]
Hi, thanks for reaching out and thanks for sharing. Just for refernce please see this: https://github.com/kedacore/keda/issues/4463
